### PR TITLE
Improve shared session tooltip

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/16485.

### Summary of Changes

Improves shared session tooltip to inform users that shared session will not work when Force HTTPS is set to Administrator only.

### Testing Instructions

Language review.

### Expected result

Used informed.

### Actual result

Used not informed.

### Documentation Changes Required

None.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_config.ini [hunk 1]
```diff
@@ -194,7 +194,7 @@ COM_CONFIG_FIELD_SESSION_HANDLER_DESC="The mechanism by which Joomla identifies
 COM_CONFIG_FIELD_SESSION_HANDLER_LABEL="Session Handler"
 COM_CONFIG_FIELD_SESSION_TIME_DESC="Auto log out a User after they have been inactive for the entered number of minutes. Do not set too high."
 COM_CONFIG_FIELD_SESSION_TIME_LABEL="Session Lifetime"
-COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site."
+COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site and that, due to the use secure cookies, shared session will not work when \"Force HTTPS\" option is set to \"Administrator Only\"."
```
Comment (richard67) [inline administrator/language/en-GB/en-GB.com_config.ini]: ... when **the** \"Force HTTPS\" option is set to \"Administrator Only\"."

I think you should add the "the".

**Comment by andrepereiradasilva** [comment]
`hound — No violations found. Woof!` ??? 💯

**Comment by Bakual** [comment]
See #16515

**Comment by richard67** [comment]
I still think that "... will not work when the \"Force HTTPS\" option is set to ..." is better English than "... will not work when \"Force HTTPS\" option is set to ...".

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 69975c27196046c5cdb8fc35a491cfe7aae57510<br /><br />The changed text is shown in the tool tip for the "Shared Sessions" value when this PR is applied.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16514">issues.joomla.org/tracker/joomla-cms/16514</a>.</sub>

File: administrator/language/en-GB/en-GB.com_config.ini [hunk 2]
```diff
@@ -194,7 +194,7 @@ COM_CONFIG_FIELD_SESSION_HANDLER_DESC="The mechanism by which Joomla identifies
 COM_CONFIG_FIELD_SESSION_HANDLER_LABEL="Session Handler"
 COM_CONFIG_FIELD_SESSION_TIME_DESC="Auto log out a User after they have been inactive for the entered number of minutes. Do not set too high."
 COM_CONFIG_FIELD_SESSION_TIME_LABEL="Session Lifetime"
-COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site."
+COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site and that, due to the use secure cookies, shared sessions will not work when the \"Force HTTPS\" option is set to \"Administrator Only\"."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: please make this a new sentence. long sentences are harder to understand and introduce translation issues. It is also enough to simply add

"This will not work when Force https is set to administrator onlly

Comment (andrepereiradasilva) [inline administrator/language/en-GB/en-GB.com_config.ini]: @brianteeman can i ask you to put a better english? thanks

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: +COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site. Shared sessions is not available when the \"Force HTTPS\" option is set to \"Administrator Only\"."

**Comment by andrepereiradasilva** [comment]
changed thanks @brianteeman

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on f5ebae42a9244f73a68ed7f8d384cc00e683ce42<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16514">issues.joomla.org/tracker/joomla-cms/16514</a>.</sub>

File: administrator/language/en-GB/en-GB.com_config.ini (hunk 2 above)
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_config.ini]: `Shared session is not` or `Shared sessions are not`?

File: administrator/language/en-GB/en-GB.com_config.ini [hunk 3]
```diff
@@ -194,7 +194,7 @@ COM_CONFIG_FIELD_SESSION_HANDLER_DESC="The mechanism by which Joomla identifies
 COM_CONFIG_FIELD_SESSION_HANDLER_LABEL="Session Handler"
 COM_CONFIG_FIELD_SESSION_TIME_DESC="Auto log out a User after they have been inactive for the entered number of minutes. Do not set too high."
 COM_CONFIG_FIELD_SESSION_TIME_LABEL="Session Lifetime"
-COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site."
+COM_CONFIG_FIELD_SHARED_SESSION_DESC="When enabled, a user's session is shared between the frontend and administrator sections of the site. Note that changing this value will invalidate all existing sessions on the site. Shared sessions is not available when the \"Force HTTPS\" option is set to \"Administrator Only\"."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: Having thought more on this it is possible that a translator will not correctly understand this - so I will have another attempt at rewording this later - please be patient and do not merge

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: When the \"Force HTTPS\" option is set to \"Administrator Only\" this is not available.

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_config.ini]: that should be easier

Comment (Quy) [inline administrator/language/en-GB/en-GB.com_config.ini]: I would rephrase it `This is not available when the "Force HTTPS" option is set to "Administrator Only".` to let the user know upfront and not later in the sentence.

**Comment by andrepereiradasilva** [comment]
ok corrected. please check

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 719fc31e4fe1bb0cca3d1f907efce76ce74e66f9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16514">issues.joomla.org/tracker/joomla-cms/16514</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 719fc31e4fe1bb0cca3d1f907efce76ce74e66f9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16514">issues.joomla.org/tracker/joomla-cms/16514</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.
